Dockerfile-alpine.template

```diff
@@ -33,10 +52,13 @@ RUN addgroup -g 1000 node \
     && ./configure \
     && make -j$(getconf _NPROCESSORS_ONLN) V= \
     && make install \
-    && apk del .build-deps \
+    && apk del .build-deps-full \
     && cd .. \
     && rm -Rf "node-v$NODE_VERSION" \
-    && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
+    && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
+  ) \
+  && rm "node-v$NODE_VERSION-linux-$ARCH-musl.tar.xz" \
```

Comment:
Needs a `-f` or the build will fail on architectures that don't have the prebuilt file from `unofficial-builds.nodejs.org`.

Reply:
Done!

Reply:
Done!